README.md

```diff
@@ -83,7 +83,7 @@ Don't forget to give the project a star! Thanks again!
 ### Running tests locally
 
 1. `npm install`
-2. `npm node`
+2. `npm run node`
```

Comment:
Can you remove 3, and change 2 to `npm run test`

Reply:
You don't want npm run node anymore? regarding npm run test, I could, but npm test is a known command, npm run node is not, LMK if you want me to edit it anyways.

Reply:
You don't need npm run node to run the unit tests